src/app/api/onboard/route.ts

```diff
@@ -0,0 +1,7 @@
+import { NextResponse } from "next/server";
+
+export async function POST(req: Request) {
+  const data = await req.json();
```

Comment:
Lets define a schema for the input JSON so that we can validate and throw errors if invalid data is submitted. You can define the schema to make all form input to be mandatory without which the schema validation fails.

See this for more details:

https://nextjs.org/docs/pages/building-your-application/data-fetching/forms-and-mutations#form-validation